resources/lang/nl/markdown.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+return [
+
+    'cheatsheet' => '
+<p>Markdown is een text-naar-HTML markup syntax for webredacteuren. Met Markdown kun je leesbare en eenvoudige platte tekst te schrijven dat geconverteerd wordt naar semantische en valide HTML.</p>
```

Comment:
Voorstel:
Markdown is een text-naar-HTML markup syntax for webredacteuren. Met Markdown kun je leesbare en eenvoudige platte tekst schrijven welke geconverteerd wordt naar semantische, valide HTML.